### PR TITLE
Styled Range Inputs

### PR DESCRIPTION
**Pull request by hobnob**
Styles range inputs to look more Amaziumy across all browsers

Example here:
http://codepen.io/hobnob/pen/RavJVv

### CODE REVIEW

File: css/form.css [hunk 1]
```diff
@@ -57,6 +57,25 @@ input[type="radio"] + label:before                          { margin:0; content:
...
 input[type="radio"] + label + small                         { margin:0; padding:0 0 0 32px; background:none; border:none; color:#E74C3C; width:auto; float:none; }
 
+/***********************************************
+Styled Range Inputs
+***********************************************/
+input[type=range]                                           { -webkit-appearance: none; width: 70%; line-height: 40px; border: 0; -moz-box-sizing: border-box; -webkit-box-sizing: border-box; box-sizing: border-box; background-color: transparent; }
+input[type=range]:focus                                     { outline: none; }
+input[type=range]::-webkit-slider-runnable-track            { width: 100%; height: 20px; cursor: pointer; background: #333; border-radius: 0; border: 1px solid #555; }
+input[type=range]::-webkit-slider-thumb                     { border: 1px solid #2ecc71; height: 40px; width: 20px; border-radius: 0; background: #2ecc71; cursor: pointer; -webkit-appearance: none; margin-top: -9.5px; }
```
Comment (catchamonkey) [inline css/form.css]: "-9.5px" really?

Comment (hobnob) [inline css/form.css]: Ah.. that's not needed any more.. removed

**Comment by catchamonkey** [comment]
How much testing have you done to ensure it plays nice with all existing variations on input placement etc?

**Comment by hobnob** [comment]
I'm using it with the site I'm doing at the moment, and there don't appear to be any adverse affects to any of the inputs that I've seen (and really, there shouldn't be, given the explicit declaration of type="range").

I have yet to cross browser test it thoroughly in the Amazium framework though. So far I've tested the stand alone code pen in IE 11, Edge, Firefox, and Chrome, but have only tested it with other components in Chrome

**Comment by catchamonkey** [comment]
Yeah it's not the new CSS that I mean would break other inputs, but just the change in styling on this input that could cause other things to be wonky if they are next to it, near it etc.

I'll leave it for @MikeBallan :)

**Comment by catchamonkey** [comment]
Closing this for now, a change is incoming and we will come back and cherry pick if needed.

Thanks!

**Pull request closed without merging**